### DISCUSSION TITLE
H3 Modbus Integration and Nathan's Energy Management Plugin

### DISCUSSION THREAD

**Discussion by xanak76** [nathanmarlor/foxess_modbus#310] [Q&A] [answered]
Dear all, after I integrated my H3 via modbus and Elfin adapter into my HA system with the great support from your side, I wonder if I can also use Nathan's Energy Management Plugin (https://github.com/nathanmarlor/foxess_em) with this configuration? When I try to configure it, the plugin does not find my Elfin adapter under modbus tcp. Thank you for any advice or support.

best regards fred.

**Comment by canton7** [comment, marked as answer]
@nathanmarlor?

You might have better luck asking on the foxess_em project. Make sure you include the error which makes you think it "does not find" the adapter (you may need to look at the logs under Settings -> System -> Logs).
